custom-fonts.md

```diff
@@ -59,6 +58,42 @@ var buttonText = const Text(
   style: textStyle,
 );
 ```
+### Fonts in package dependencies {#from-packages}
+
+To use a font family defined in a package, the `package` argument must be provided. For instance, suppose the font
+declaration above is in the `pubspec.yaml` of a package named `my_package` which the app depends on. Then creating the
+TextStyle is done as follows:
+
+```dart
+const textStyle = const TextStyle(
+  fontFamily: 'Raleway',
+  package: 'my_package',
+);
+```
+
+This is also how the package itself should create the style.
```

Comment:
I was reading this doc and had some questions I wasn't clear on. When referring to the 'package itself', is it saying that if a package was to internally consume the font it bundled, it should still say TextStyle(fontFamily: 'name', package: 'self_package_name')? or did you mean something else?

Reply:
Yes, that is what I mean.

Reply:
Please make sure the text (on the Web site and in the API docs since we seem to have the same text in both places) is updated to clarify the issues @xster raised.